presto-benchto-benchmarks/pom.xml

```diff
@@ -26,6 +26,18 @@
             <groupId>com.teradata.benchto</groupId>
             <artifactId>benchto-driver</artifactId>
             <version>0.4</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.hibernate</groupId>
+                    <artifactId>hibernate-validator</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
+            <artifactId>hibernate-validator</artifactId>
+            <version>6.2.0.Final</version>
```

Comment:
As per maven repo, looks like even `6.2.0.Final`l has CVEs? https://mvnrepository.com/artifact/org.hibernate.validator/hibernate-validator/6.2.0.Final

Reply:
@imsayari404 Java upgrade work is already in progress.